src/options.rs

```diff
@@ -108,3 +101,21 @@ impl TryFrom<&Args> for Options {
         })
     }
 }
+
+fn build_glob_set(glob_set: &Vec<String>) -> Result<Option<GlobSet>>{
+    let globset = if glob_set.is_empty() {
```

Comment:
In Rust it would you don't need to assign to a variable just to return it; just let the `if` be the single expression in this function. (Clippy might say this?)

Reply:
good point, done!